index.html

```diff
@@ -0,0 +1,162 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+
+    <link rel="stylesheet" href="style.css" />
+</head>
+<body>
+    <section>
+    <div class="nav_bar">
+        <div class="logo">Davis Nyale</div>
+        <div class="image">
+            <img src="./images/navbar.png" alt="navbar">
+        </div>
+    </div>
+    <div class="body_section">
+        <div class="text">
+            <h1>Davis Nyale</h1>
+            <h2>Software Developer</h2>
+            <h4>I can help you build a product ,<br>
+                 feature or website Look through <br>
+                 some of my work and experience! If<br>
+                  you like what you see and have a<br>
+                   project you need coded,  don’t<br>
+                   hestiate to contact me.
+                </h4>
+                <img src="./images/Vector 4.png">
+                <div class="languages_menu">
+                    <ul>
+                        <li class="heading">languages</li>
+                        <li>Javascript</li>
+                        <li>Ruby</li>
+                        <li>Html</li>
+                        <li>css</li>
+                    </ul>
+                    <ul >
+                        <li class="heading">Frameworks</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby on Rails</li>
+                        <li>RsPec</li>
+                        <li>CabyBara</li>
+                        <li>Selenium</li>
+                    </ul>
+                    <ul >
+                        <li class="heading">Skills</li>
+                        <li>Codekit</li>
+                        <li>Github</li>
+                        <li>Codepen</li>
+                        <li>Gitlab</li>
+                        <li>Terminal</li>
+                    </ul>
+                </div>
+        </div>
+        <img src="./images/Vector 4.png">
+        <div class="social_media">
+            <img src="./images/github.png" alt="github">
+            <img src="./images/Linkedin icon.png" alt="linkedin">
+            <img src="./images/angellist.png" alt="angelist">
+            <img src="./images/twitter.png" alt="twitter">
+            <img src="./images/medium.png" alt="medium">
+        </div>
```

Comment:
- [ ] Please change the main wrapper to a ul tag and wrap all social media icons in an li and a tag, it should look like this: 
`<li><a href="#"><img src="./images/medium.png" alt="medium"></a></li>`

- this will make your links clickable and more semantic 👍🏿 

- [ ] Kindly add the appropriate padding to each link so it looks like the Figma design